users/serializers.py

```diff
@@ -63,36 +81,44 @@ class Meta:
             'email',
             'first_name',
             'last_name',
-            'date_of_birth',
-            'phone_number',
-            'country',
             'user_type',
-            'is_staff',
-            'is_superuser',
-            'is_active',
         )
 
 
 class CustomRegisterSerializer(serializers.Serializer):
     email = serializers.EmailField(required=allauth_settings.EMAIL_REQUIRED)
-    first_name = serializers.CharField(required=False, write_only=True)
-    last_name = serializers.CharField(required=False, write_only=True)
-    password = serializers.CharField(required=True, write_only=True)
-    password_confirmation = serializers.CharField(required=True, write_only=True)
+    first_name = serializers.CharField(
+        required=False,
+        allow_blank=True,
+        write_only=True,
+    )
+    last_name = serializers.CharField(
+        required=False,
+        allow_blank=True,
+        write_only=True,
+    )
+    password1 = serializers.CharField(required=True, write_only=True, style={"input_type":"password"})
+    password2 = serializers.CharField(required=True, write_only=True, style={"input_type":"password"})
```

Comment:
Why did you change it from `password` and `password_confirmation` to `password1` and `password2`?

Reply:
When I change to for example password and password_confirmation it doesn't work. Registration pass but user password is not set and user cannot log in. For now I only added label for these fields to be more specific